morph/protocol.privnet.yml

```diff
@@ -32,6 +32,8 @@ ApplicationConfiguration:
     EnableCORSWorkaround: false
     MaxGasInvoke: 15
     Port: 30333
+    SessionEnabled: true
+    SessionExpirationTime: 30
```

Comment:
How about enabling `SessionBackedByMPT` then? We use bolt here, it performs re-mmapings periodically, and it may be critical for the blockchain persist cycles. I like `SessionBackedByMPT` setting, it has a separate state that doesn't affect the rest of blockchain functionality.

Reply:
Shouldn't make a lot of difference for AIO, it grows slowly and we're using it from "trusted" software, that shouldn't leave session open for long.